src/nplinker/pairedomics/podp_antismash_downloader.py

```diff
@@ -214,16 +294,16 @@ def _get_antismash_db_page(genome_obj):
 
             if link is not None:
                 logger.info(
-                    'antiSMASH lookup succeeded! Filename is {}'.format(
-                        link['href']))
+                    f"antiSMASH lookup succeeded! Filename is {link['href']}")
                 # save with the .1 suffix if that worked
-                genome_obj.resolved_id = accession
+                genome_obj.resolved_refseq_id = accession
                 return link['href']
 
     return None
 
 
-def _get_antismash_zip_data(accession_id, filename, local_path):
+def _get_antismash_zip_data(accession_id: str, filename: str,
```

Comment:
This `_get_antismash_zip_data` method is trying to download antismash zip data. Please try to refactor its code using the existing  [utils/download_url](https://github.com/NPLinker/nplinker/blob/5084cc548bd6ca6351591acf5a544a9dd97c722d/src/nplinker/utils.py#L163) function.